EIPS/eip-nft_hierarchy.md

```diff
@@ -0,0 +1,247 @@
+---
+eip: <TBA>
```

Comment:
```suggestion
eip: 7510
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.